[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:02:31,421 ERROR [http-nio-53050-exec-1][GlobalExceptionHandler.java:47] - 【系统异常】null
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:03:34,455 ERROR [http-nio-53050-exec-3][GlobalExceptionHandler.java:47] - 【系统异常】null
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:27:50,651 ERROR [hmilyTransaction-hmily-transaction-self-recovery-72][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:28:15,498 ERROR [hmilyTransaction-hmily-transaction-self-recovery-18][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:28:15,514 ERROR [hmilyTransaction-hmily-transaction-self-recovery-36][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:28:24,743 ERROR [hmilyTransaction-hmily-transaction-self-recovery-54][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:28:50,660 ERROR [hmilyTransaction-hmily-transaction-self-recovery-72][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:29:09,197 ERROR [http-nio-53050-exec-5][GlobalExceptionHandler.java:47] - 【系统异常】null
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:29:15,510 ERROR [hmilyTransaction-hmily-transaction-self-recovery-18][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:29:15,522 ERROR [hmilyTransaction-hmily-transaction-self-recovery-36][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:29:24,756 ERROR [hmilyTransaction-hmily-transaction-self-recovery-54][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:29:50,676 ERROR [hmilyTransaction-hmily-transaction-self-recovery-72][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:30:15,514 ERROR [hmilyTransaction-hmily-transaction-self-recovery-18][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:30:15,529 ERROR [hmilyTransaction-hmily-transaction-self-recovery-36][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:30:24,771 ERROR [hmilyTransaction-hmily-transaction-self-recovery-54][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:30:50,684 ERROR [hmilyTransaction-hmily-transaction-self-recovery-72][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:31:15,532 ERROR [hmilyTransaction-hmily-transaction-self-recovery-18][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:31:15,544 ERROR [hmilyTransaction-hmily-transaction-self-recovery-36][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:31:24,788 ERROR [hmilyTransaction-hmily-transaction-self-recovery-54][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:31:50,690 ERROR [hmilyTransaction-hmily-transaction-self-recovery-72][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:32:15,541 ERROR [hmilyTransaction-hmily-transaction-self-recovery-18][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:32:15,557 ERROR [hmilyTransaction-hmily-transaction-self-recovery-36][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:32:24,794 ERROR [hmilyTransaction-hmily-transaction-self-recovery-54][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:32:50,693 ERROR [hmilyTransaction-hmily-transaction-self-recovery-72][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:33:15,554 ERROR [hmilyTransaction-hmily-transaction-self-recovery-18][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:33:15,569 ERROR [hmilyTransaction-hmily-transaction-self-recovery-36][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:33:24,808 ERROR [hmilyTransaction-hmily-transaction-self-recovery-54][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:33:50,709 ERROR [hmilyTransaction-hmily-transaction-self-recovery-72][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:34:15,563 ERROR [hmilyTransaction-hmily-transaction-self-recovery-18][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:34:15,593 ERROR [hmilyTransaction-hmily-transaction-self-recovery-36][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:34:20,911 ERROR [hmilyTransaction-hmily-log-disruptor12-73][JdbcCoordinatorRepository.java:257] - executeUpdate-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:34:21,402 ERROR [hmilyTransaction-hmily-log-disruptor12-92][JdbcCoordinatorRepository.java:257] - executeUpdate-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:34:21,403 ERROR [hmilyTransaction-hmily-log-disruptor12-92][JdbcCoordinatorRepository.java:257] - executeUpdate-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:34:21,417 ERROR [hmilyTransaction-hmily-log-disruptor12-92][JdbcCoordinatorRepository.java:257] - executeUpdate-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:34:21,502 ERROR [hmilyTransaction-hmily-log-disruptor12-92][JdbcCoordinatorRepository.java:257] - executeUpdate-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:34:24,816 ERROR [hmilyTransaction-hmily-transaction-self-recovery-54][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:34:50,719 ERROR [hmilyTransaction-hmily-transaction-self-recovery-72][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:35:15,579 ERROR [hmilyTransaction-hmily-transaction-self-recovery-18][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:35:15,609 ERROR [hmilyTransaction-hmily-transaction-self-recovery-36][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:35:24,817 ERROR [hmilyTransaction-hmily-transaction-self-recovery-54][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:35:50,724 ERROR [hmilyTransaction-hmily-transaction-self-recovery-72][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:36:15,593 ERROR [hmilyTransaction-hmily-transaction-self-recovery-18][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:36:15,624 ERROR [hmilyTransaction-hmily-transaction-self-recovery-36][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:36:20,943 ERROR [hmilyTransaction-hmily-transaction-self-recovery-91][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:36:24,824 ERROR [hmilyTransaction-hmily-transaction-self-recovery-54][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:36:50,747 ERROR [hmilyTransaction-hmily-transaction-self-recovery-72][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:37:15,607 ERROR [hmilyTransaction-hmily-transaction-self-recovery-18][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:37:15,637 ERROR [hmilyTransaction-hmily-transaction-self-recovery-36][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:37:20,955 ERROR [hmilyTransaction-hmily-transaction-self-recovery-91][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:37:24,831 ERROR [hmilyTransaction-hmily-transaction-self-recovery-54][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:37:50,765 ERROR [hmilyTransaction-hmily-transaction-self-recovery-72][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:38:15,617 ERROR [hmilyTransaction-hmily-transaction-self-recovery-18][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:38:15,646 ERROR [hmilyTransaction-hmily-transaction-self-recovery-36][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:38:20,958 ERROR [hmilyTransaction-hmily-transaction-self-recovery-91][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:38:24,840 ERROR [hmilyTransaction-hmily-transaction-self-recovery-54][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:38:50,766 ERROR [hmilyTransaction-hmily-transaction-self-recovery-72][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:39:15,622 ERROR [hmilyTransaction-hmily-transaction-self-recovery-18][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:39:15,655 ERROR [hmilyTransaction-hmily-transaction-self-recovery-36][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:39:20,974 ERROR [hmilyTransaction-hmily-transaction-self-recovery-91][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:39:24,850 ERROR [hmilyTransaction-hmily-transaction-self-recovery-54][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:39:50,777 ERROR [hmilyTransaction-hmily-transaction-self-recovery-72][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:40:15,623 ERROR [hmilyTransaction-hmily-transaction-self-recovery-18][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:40:15,669 ERROR [hmilyTransaction-hmily-transaction-self-recovery-36][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:40:20,990 ERROR [hmilyTransaction-hmily-transaction-self-recovery-91][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:40:24,862 ERROR [hmilyTransaction-hmily-transaction-self-recovery-54][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:40:50,787 ERROR [hmilyTransaction-hmily-transaction-self-recovery-72][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:41:15,637 ERROR [hmilyTransaction-hmily-transaction-self-recovery-18][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:41:15,684 ERROR [hmilyTransaction-hmily-transaction-self-recovery-36][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:41:21,006 ERROR [hmilyTransaction-hmily-transaction-self-recovery-91][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:41:24,866 ERROR [hmilyTransaction-hmily-transaction-self-recovery-54][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:41:50,796 ERROR [hmilyTransaction-hmily-transaction-self-recovery-72][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:42:15,644 ERROR [hmilyTransaction-hmily-transaction-self-recovery-18][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:42:15,692 ERROR [hmilyTransaction-hmily-transaction-self-recovery-36][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:42:21,017 ERROR [hmilyTransaction-hmily-transaction-self-recovery-91][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:42:24,881 ERROR [hmilyTransaction-hmily-transaction-self-recovery-54][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:42:50,799 ERROR [hmilyTransaction-hmily-transaction-self-recovery-72][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:42:59,984 ERROR [http-nio-53050-exec-9][GlobalExceptionHandler.java:47] - 【系统异常】null
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:43:15,652 ERROR [hmilyTransaction-hmily-transaction-self-recovery-18][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:43:15,699 ERROR [hmilyTransaction-hmily-transaction-self-recovery-36][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:43:21,022 ERROR [hmilyTransaction-hmily-transaction-self-recovery-91][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:43:24,896 ERROR [hmilyTransaction-hmily-transaction-self-recovery-54][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:43:50,806 ERROR [hmilyTransaction-hmily-transaction-self-recovery-72][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:44:15,658 ERROR [hmilyTransaction-hmily-transaction-self-recovery-18][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:44:15,705 ERROR [hmilyTransaction-hmily-transaction-self-recovery-36][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:44:21,025 ERROR [hmilyTransaction-hmily-transaction-self-recovery-91][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:44:24,901 ERROR [hmilyTransaction-hmily-transaction-self-recovery-54][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:44:50,810 ERROR [hmilyTransaction-hmily-transaction-self-recovery-72][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:45:15,660 ERROR [hmilyTransaction-hmily-transaction-self-recovery-18][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:45:15,722 ERROR [hmilyTransaction-hmily-transaction-self-recovery-36][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:45:21,035 ERROR [hmilyTransaction-hmily-transaction-self-recovery-91][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:45:24,903 ERROR [hmilyTransaction-hmily-transaction-self-recovery-54][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:45:50,813 ERROR [hmilyTransaction-hmily-transaction-self-recovery-72][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:46:15,667 ERROR [hmilyTransaction-hmily-transaction-self-recovery-18][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:46:15,729 ERROR [hmilyTransaction-hmily-transaction-self-recovery-36][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:46:21,041 ERROR [hmilyTransaction-hmily-transaction-self-recovery-91][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:46:24,907 ERROR [hmilyTransaction-hmily-transaction-self-recovery-54][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:46:50,825 ERROR [hmilyTransaction-hmily-transaction-self-recovery-72][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:47:15,678 ERROR [hmilyTransaction-hmily-transaction-self-recovery-18][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:47:15,736 ERROR [hmilyTransaction-hmily-transaction-self-recovery-36][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:47:21,043 ERROR [hmilyTransaction-hmily-transaction-self-recovery-91][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:47:24,921 ERROR [hmilyTransaction-hmily-transaction-self-recovery-54][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
[wanxinp2p-consumer-service][${env:SERVER_PORT}] 2021-10-18 20:47:50,831 ERROR [hmilyTransaction-hmily-transaction-self-recovery-72][JdbcCoordinatorRepository.java:298] - executeQuery-> Table 'p2p_undo_log.hmily_consumer_service' doesn't exist
